department-of-veterans-affairs/caseflow
# adds real appeal data to endpoint

Connects https://github.com/department-of-veterans-affairs/caseflow/issues/6361
Connects https://github.com/department-of-veterans-affairs/caseflow/issues/6360


# Test plan
1. `curl localhost:3000/idt/api/v1/token`
1. note one time key
1. open `localhost:3000/idt/auth?one_time_key=KEY` in the browser while logged in as attorney user
1. observe key validation succeeding
1. `curl -H "TOKEN: token" localhost:3000/idt/api/v1/appeals`
1. Verify that cases returned match cases in the attorney's queue, if all those cases are activated
1. `curl -H "FILE: some_file_number" -H "TOKEN: token" localhost:3000/idt/api/v1/appeals`
1. Verify that the cases returned match those returned by Case Search in the UI, if all those cases are activated
